packages/base/src/signal-manager.ts

```diff
@@ -3,6 +3,7 @@ import { EventEmitter } from 'events';
 export declare interface SignalManager {
 
     fireTooltipSignal(tooltip: { [key: string]: string }): void;
+    fireEventClickSignal(payload: {[key: string]: number }): void;
```

Comment:
we should send the keys and value as strings and the receiving side should convert to the number. Later on we'll add other key/value pairs to sync on.